src/t_set.cpp

```diff
@@ -325,6 +348,30 @@ void saddCommand(client *c) {
     addReplyLongLong(c,added);
 }
 
+void saddintCommand(client *c) {
+    robj *set;
+    int j, added = 0;
+
+    set = lookupKeyWrite(c->db,c->argv[1]);
+    if (checkType(c,set,OBJ_SET)) return;
+
+    if (set == NULL) {
+        //set = setTypeCreate(szFromObj(c->argv[2]));
```

Comment:
Remove commented out code

Reply:
I kow it's hard, but forcing Werror and O3 while cimpiling, it keep our code clean.

Reply:
Re Werror, KeyDB's policy is to have no warnings during build, however because we ship source code we get compiled on all sorts of compilers and different versions of GCC all with their own warnings.  Therefore we don't enable this flag as it causes problems for downstream users.  If you look in our CI file however you'll notice we do use this flag where its in a controlled situation.

Re -O3 in benchmarks it makes no perceptible difference and introduces a higher risk for bugs.